utils/combined-translation-keys.txt

```diff
@@ -158,5 +158,5 @@ q1:
 q3:                                                                                                         // plotly.js: traces/box/calc.js:131
 source:                                                                                                     // plotly.js: traces/sankey/plot.js:140
 target:                                                                                                     // plotly.js: traces/sankey/plot.js:141
-trace                                                                                                       // plotly.js: plots/plots.js:439
+trace                                                                                                       // plotly.js: plots/plots.js:442
 upper fence:                                                                                                // plotly.js: traces/box/calc.js:135
```

Comment:
oh right, the css changes change the line references of the translation strings, great, thanks for updating!